pkg/kubelet/pleg/generic_test.go

```diff
@@ -479,7 +480,7 @@ func TestRelistWithReinspection(t *testing.T) {
 	actualEvents := getEventsFromChannel(ch)
 	actualStatus, actualErr := pleg.cache.Get(podID)
 	assert.Equal(t, goodStatus, actualStatus)
-	assert.Equal(t, nil, actualErr)
+	require.NoError(t, actualErr)
```

Comment:
Can you explain why you've changed `assert` to `require` in some cases? Other than that the PR looks pretty simple and straightforward to me.

Reply:
/lgtm

Reply:
There are still places where you `assert` on an error, and then `defer` the use of the other return, e.g. `Close()`